modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view.jsp

```diff
@@ -351,6 +351,12 @@ DLViewDisplayContext dlViewDisplayContext = new DLViewDisplayContext(dlAdminDisp
 			/>
 		</div>
 
+		<div>
```

Comment:
[PROPOSAL not mandatory]

Hey @ambrinchaudhary 
Shouldn't this be under feature flag?